lib/suspenders/adapters/heroku.rb

```diff
@@ -0,0 +1,67 @@
+module Suspenders
+  module Adapters
+    class Heroku
+      def initialize(app_builder)
+        @app_builder = app_builder
+      end
+
+      def set_heroku_remotes
+        remotes = <<-SHELL.strip_heredoc
+
+          # Set up the staging and production apps.
+          #{join_heroku_app('staging')}
+          #{join_heroku_app('production')}
+        SHELL
+
+        app_builder.append_file "bin/setup", remotes
+      end
+
+      def setup_heroku_specific_gems
```

Comment:
This should really be named `set_up_heroku_specific_gems`. I know, grammar nazi :)